server/chat-commands/admin.ts

```diff
@@ -1015,10 +1015,10 @@ export const commands: ChatCommands = {
 			return 'p3';
 		}
 		function getPokemon(input: string) {
-			if (/^[0-9]+$/.test(input)) {
+			if (!isNaN(parseInt(input))) {
```

Comment:
```suggestion
			if (/^[0-9]+$/.test(input.trim())) {
```

I'd allow whitespace this way, because the way you have it implemented, it will match anything starting with a digit `0-9`, rather than just numbers.

Reply:
(That's why I had the regex in the first place.)